template_scripts/packages_fc.list

```diff
@@ -65,9 +65,11 @@ zip
 --exclude=gnome-user-docs
 --exclude=grubby
 --exclude=gssproxy
+--exclude=hplip
```

Comment:
This will break printing on HP devices. I don't think we should do that.
Previously excluding avahi was more or less harmless (it is useless behind sys-net+sys-firewall), but if it isn't the case anymore, then we should stop excluding it.